src/rpc/calls/work_validate.rs

```diff
@@ -0,0 +1,44 @@
+use crate::blocks::BlockHash;
+use crate::rpc::calls::{as_str, from_str};
+use crate::rpc::client::{RPCClient, RPCRequest};
+use crate::{Difficulty, Result, Work};
+use async_trait::async_trait;
+use clap::Clap;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clap)]
+pub struct WorkValidateRequest {
+    pub hash: BlockHash,
+    pub work: Work,
+}
+
+#[async_trait]
```

Comment:
What does this do?

Reply:
There's an issue with traits and async, see https://github.com/dtolnay/async-trait